index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: Setup and Mobile First Web App</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+
+    <div class="wrapper">
+        <header class="header">
+            <a href="./index.html" class="my-logo">My Logo</a>
+            <nav class="navbar">
+                <ul>
+                    <li class="menu-bar">
+                        <a href="#" class="bar"><i class="fa fa-bars"></i></a>
+                    </li>
+                </ul>
```

Comment:
- [ ] The hamburger button should be around `<button>` tag since in the future you will implement functionality to open the mobile menu through it and then the mobile menu will have a list of links. Kindly use the correct tag for it.
 - [ ] The hamburger button is not correctly aligned with the logo on the left. It has a little offset to the bottom. Kindly adjust its position.
 -
<img width="441" alt="image" src="https://user-images.githubusercontent.com/61919617/186202051-f292c486-42ef-451e-bd8a-3b6fec7509c1.png">